incubator/vault-ui/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
We've started namespacing templates. See this issue and apply the same to your PR: #1785
Eventually, Helm create will do this for you in a future release. See kubernetes/helm#2845

`vault-ui.name`, `vault-ui.fullname`